pkgs/development/python-modules/jedi/default.nix

```diff
@@ -22,9 +24,12 @@ buildPythonPackage rec {
     LC_ALL="en_US.UTF-8" py.test test
   '';
 
-  # tox required for tests: https://github.com/davidhalter/jedi/issues/808
+  # A few tests failed because of having to access home directory.
+  # Therefore disable the tests for now.
   doCheck = false;
 
+  pythonImportsCheck = [ "jedi" ];
+
   meta = with lib; {
     homepage = "https://github.com/davidhalter/jedi";
     description = "An autocompletion tool for Python that can be used for text editors";
```

Comment:
You want to add yourself as a maintainer?

Reply:
Yes I do want to add myself as a maintainer - will do that!